lib/main.js

```diff
@@ -235,10 +261,19 @@ export default {
       }
     }
 
-    const linter = new Linter(Object.assign({
-      formatter: 'json',
-      rulesDirectory,
-    }, options));
+    const program = await this.getProgram(Linter, configurationPath);
+    let linter;
+    if (program) {
```

Comment:
If the `if/else` needed? Since passing null/undefined is effectively the same as not passing at all